package.json

```diff
@@ -174,7 +174,7 @@
     "@unstoppabledomains/resolution": "^1.17.0",
     "axios": "0.21.1",
     "bignumber.js": "9.0.1",
-    "bnc-onboard": "~1.28.0",
+    "bnc-onboard": "~1.29.0",
```

Comment:
https://github.com/blocknative/onboard/releases/tag/1.29.0

It fixes derivation path validation for ledger, I thought I'll include it there because the testing will be done for all wallets